src/rgw/setup/rgw.py

```diff
@@ -273,7 +263,6 @@ def upgrade(conf: MappedConf, changeset_path: str):
         except Exception as e:
             raise SetupError(errno.EINVAL, f'Upgrade failed with error: {e}')
 
-        Log.info('Upgrade phase completed.')
```

Comment:
'completed' phase logs are rather useful, instead we can think of removing phase 'started' logs, where ever exists. 
Phases which are not implemented yet, like reset, test, can be avoided with 'completed' logs though, which you have done above

Reply:
We have added a log statement for **'{phase_name} phase completed'** in rgw_setup.py file , So here it is not required.

Reply:
yes.